licenses-report.txt

```diff
@@ -50,210 +163,430 @@ The following software have components provided under the terms of this license:
 - jersey-container-servlet (from git://java.net/jersey~code/project/jersey-container-servlet)
-- jersey-media-json-jackson (from git://java.net/jersey~code/project/jersey-media-json-jackson)
+- jersey-container-servlet-core (from git://java.net/jersey~code/project/jersey-container-servlet-core)
+- jersey-core-client (from git://java.net/jersey~code/jersey-client)
+- jersey-core-common (from )
+- jersey-core-server (from git://java.net/jersey~code/jersey-server)
+- jersey-ext-bean-validation (from )
+- jersey-ext-metainf-services (from )
+- jersey-inject-hk2 (from )
+- jersey-media-jaxb (from )
+- jersey-test-framework-core (from )
+- jersey-test-framework-provider-inmemory (from )
 
 ========================================================================
 GPL-3.0-only
 ========================================================================
 The following software have components provided under the terms of this license:
 
+- Expression Language 3.0 (from https://projects.eclipse.org/projects/ee4j.el)
+- Java Servlet API (from https://projects.eclipse.org/projects/ee4j.servlet)
+- MIME streaming extension (from http://mimepull.java.net)
 - OSGi resource locator (from )
-- javax.ws.rs-api (from http://jax-rs-spec.java.net)
+- jakarta.annotation-api (from https://projects.eclipse.org/projects/ee4j.ca)
+- javax.ws.rs-api (from https://github.com/eclipse-ee4j/jaxrs-api)
 
 ========================================================================
 ISC
 ========================================================================
 The following software have components provided under the terms of this license:
 
+- jbcrypt (from http://www.mindrot.org/)
 - jbcrypt (from http://www.mindrot.org/)
 
 ========================================================================
 JSON
 ========================================================================
 The following software have components provided under the terms of this license:
 
+- JSON in Java (from https://github.com/douglascrockford/JSON-java)
 - config (from https://github.com/typesafehub/config)
 
 ========================================================================
 LGPL-2.1-only
 ========================================================================
 The following software have components provided under the terms of this license:
 
+- Commons Lang (from http://commons.apache.org/lang/)
+- Dropwizard Logging Support (from )
+- FindBugs-Annotations (from http://findbugs.sourceforge.net/)
+- HPPC Collections (from http://labs.carrotsearch.com)
+- Hibernate Validator Engine (from )
+- JBoss Logging 3 (from http://www.jboss.org)
+- Java Native Access (from https://github.com/java-native-access/jna)
+- Java Native Access (from https://github.com/java-native-access/jna)
+- Java Native Access (from https://github.com/java-native-access/jna)
+- Java Native Access Platform (from https://github.com/java-native-access/jna)
+- Java Native Access Platform (from https://github.com/java-native-access/jna)
```

Comment:
another duplicate?

Is this auto-generated?

Reply:
Yes and I thought I got them all. Guess not :)